README.md

```diff
@@ -311,8 +311,8 @@ client = hazelcast.HazelcastClient(config)
 
 ---
 
-If you run the Hazelcast IMDG members in a different server than the client, you most probably have configured the members' ports and cluster
-names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
+If you run the Hazelcast IMDG members on a different server than the client, you most probably have configured the members' ports and cluster
+names as explained in the previous section. If you did, then you need to make match those changes to the network settings of your client.
```

Comment:
If you run the Hazelcast IMDG members on a different server than the client, you most probably have configured the members' ports and cluster
names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
names as explained in the previous section. If you did, then you need to match those changes to the network settings of your client.